src/frontend/src/views/Upload/components/common/constants.ts

```diff
@@ -10,6 +10,16 @@ export const HEADERS_TO_METADATA_KEYS = invert(
   SAMPLE_UPLOAD_METADATA_KEYS_TO_HEADERS
 ) as Dictionary<keyof SampleUploadTsvMetadata>;
 
+export const NEXTSTRAIN_FORMAT_HEADERS_TO_METADATA_KEYS: Record<
+  string,
+  string
+> = {
+  date: "collectionDate",
+  location: "collectionLocation",
+  gisaid_epi_isl: "publicId",
+  strain: "sampleId",
```

Comment:
I'm surprised this isn't lineage or something. What's going on here?

Reply:
I was just following the ticket description, wondering if @sidneymbell can double check this?

Reply:
It sounds like this column is called `sampleId` in our template, but it is called `strain` in the Nextstrain metadata, so we're mapping both to `"sampleId"`?

Reply:
yep!